antarest/study/storage/rawstudy/model/filesystem/config/links.py

```diff
@@ -66,6 +67,74 @@ class TransmissionCapacity(EnumIgnoreCase):
     ENABLED = "enabled"
 
 
+class LinkStyle(EnumIgnoreCase):
+    """
+    Enum representing the style of a link in a network visualization.
+
+    Attributes:
+        DOT: Represents a dotted line style.
+        PLAIN: Represents a solid line style.
+        DASH: Represents a dashed line style.
+        DOT_DASH: Represents a line style with alternating dots and dashes.
+    """
+
+    DOT = "dot"
+    PLAIN = "plain"
+    DASH = "dash"
+    DOT_DASH = "dotdash"
+    OTHER = "other"
+
+
+class FilterOption(EnumIgnoreCase):
+    """
+    Enum representing the time filter options for data visualization or analysis in Antares Web.
+
+    Attributes:
+        HOURLY: Represents filtering data by the hour.
+        DAILY: Represents filtering data by the day.
+        WEEKLY: Represents filtering data by the week.
+        MONTHLY: Represents filtering data by the month.
+        ANNUAL: Represents filtering data by the year.
+    """
+
+    HOURLY = "hourly"
+    DAILY = "daily"
+    WEEKLY = "weekly"
+    MONTHLY = "monthly"
+    ANNUAL = "annual"
+
+
+def validate_filters(
+    filter_value: t.Union[t.List[FilterOption], str], enum_cls: t.Type[FilterOption]
+) -> t.List[FilterOption]:
+    if filter_value is not None and isinstance(filter_value, str):
+        filter_accepted_values = [e for e in enum_cls]
+
+        options = filter_value.replace(" ", "").split(",")
+
+        invalid_options = [opt for opt in options if opt not in filter_accepted_values]
+        if invalid_options:
+            raise LinkValidationError(
+                f"Invalid value(s) in filters: {', '.join(invalid_options)}. "
+                f"Allowed values are: {', '.join(filter_accepted_values)}."
+            )
+
+        return [enum_cls(opt) for opt in options]
+
+    return filter_value
+
+
+def join_with_comma(values: t.List[FilterOption]) -> str:
```

Comment:
Is it really useful ? Also, it also lowers the values so we could add this inside the function name. 
I think we should remove it as it's only used once

Reply:
I like it to have a method even if its use one time, its clear at first sight the transformation done in the serialization thanks to the method name. We can discuss it

Reply:
Why not it doesn't really matter